static/css/misc.less

```diff
@@ -331,10 +331,10 @@
             bottom: 50%;
             text-align: right;
         }
-        
+
         &__before {
             padding-left: 1.25em;
-            
```

Comment:
Do we have a policy about white space? If not we should probably add that to the Front End discussion about linting code. 

*Also, do you have `trim_trailing_white_space_on_save` option turned on in ST3 or equivalent IDE? It automated removing all this extra whitespace